models/classes/class.ItemsService.php

```diff
@@ -257,7 +257,15 @@ public function getItemContent(core_kernel_classes_Resource $item, $lang = ''){
             if($itemContents->count() > 0){
                 $itemContent = $itemContents->get(0);
             }
-
+            //if no itemContent is set get the default one and copy it into a new repository
+            else{
+                $itemContents = $item->getPropertyValuesCollection($this->itemContentProperty);
```

Comment:
Redundant call here can you provide a correction here to avoid the service to do twice the same call